<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>358</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft JhengHei</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
      </font>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="title" stdset="0">
      <string>SSS</string>
     </property>
     <widget class="QWidget" name="findTab">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Find String&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="statusTip">
       <string extracomment="Find String of this doc"/>
      </property>
      <property name="accessibleName">
       <string>Find Find</string>
      </property>
      <attribute name="title">
       <string>Find</string>
      </attribute>
      <widget class="QLabel" name="targetLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Target to Find</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="findNextOneButton">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>20</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Find Next One</string>
       </property>
      </widget>
      <widget class="QPushButton" name="countButton">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>70</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Counts</string>
       </property>
      </widget>
      <widget class="QPushButton" name="cancelButton">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>170</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>191</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>SettingGroupBox</string>
       </property>
       <widget class="QCheckBox" name="findReverseCk">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Reverse</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="matchWholeWordCk">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Match Whole Word</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="matchCaseCk">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Match Case</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="showAllMatchesButon">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>120</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Show All Matches</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="replaceTab">
      <attribute name="title">
       <string>Replace</string>
      </attribute>
      <widget class="QPushButton" name="findNextOneButton_2">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>20</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Find Next One</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="targetLabel_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Target to Find</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>191</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>SettingGroupBox</string>
       </property>
       <widget class="QCheckBox" name="findReverseCk_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Reverse</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="matchWholeWordCk_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Match Whole Word</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="matchCaseCk_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Match Case</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="replaceLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Replace To</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="cancelButton_2">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>170</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="replaceButton">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>70</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
      <widget class="QPushButton" name="replaceAllButton">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>120</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="fileFindTab">
      <attribute name="title">
       <string>File Find</string>
      </attribute>
      <widget class="QLabel" name="targetLabel_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Target to Find</string>
       </property>
      </widget>
      <widget class="QLabel" name="replaceLabel_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Replace To</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_4">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_5">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="fileTypeLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>File Type</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_6">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>120</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="directoryLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Directory</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_7">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>170</y>
         <width>281</width>
         <height>31</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>170</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="findAllButton">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>20</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Find All</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QLabel" name="statusLabel">
     <property name="font">
      <font>
       <family>Microsoft JhengHei</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
